tachyon/zk/plonk/halo2/optimizations.md

```diff
@@ -89,3 +89,34 @@ Considering the memory used by polynomials is typically high, reducing the numbe
 [saving 3-h]: https://github.com/scroll-tech/halo2/blob/1070391642dd64b2d68b47ec246cba9e35bd3c15/halo2_proofs/src/plonk/keygen.rs#L566-L592
 [saving 4-t]: https://github.com/kroma-network/tachyon/pull/288/commits/44e089f5f0a83938e15b210af921c97d7d0868cf
 [saving 4-h]: https://github.com/scroll-tech/halo2/blob/1070391642dd64b2d68b47ec246cba9e35bd3c15/halo2_proofs/src/poly/domain.rs#L182
+
+## 6. Benchmark for each circuit
+
+We provide benchmarks of Tachyon and Halo2 for three main circuits by [Privacy-scaling-explorations](https://github.com/privacy-scaling-explorations/zkevm-circuits) and [Scroll](https://github.com/scroll-tech/zkevm-circuits).
+
+- Tachyon commit hash: 48891d59
+- halo2 commit hash: 0918c29e
+- zkevm-circuit commit hash: ea2c2102
+
+And the following data was measured on the AWS EC2 instance (name: r6i.8xlarge, vCPU: 32, RAM: 256GB).
+
```

Comment:
Could you add something like "The Greek symbols such as "theta" and "beta" are adopted from https://zcash.github.io/halo2/design/proving-system.html#tldr."?